MSVC: toupper is not a member of std

**I'm submitting a ...**
  - [X] bug report
  - [ ] feature request
  - [ ] support request => **Please do not submit support request here**, see note at the top of this template.

**Describe the issue**
Using MSVC 14.0 (2015), I receive the following error during compilation:
`error C2039: 'toupper': is not a member of 'std'`

**What is the current behavior?**
Compilation error:
`error C2039: 'toupper': is not a member of 'std'`

**What is the expected behavior?**
Compile without errors

**Please tell us about your environment:**
 - OS: Windows 10
 - JSBSim: v1.2.0
 - Language: C++

**Other information**
The bug is simple to resolve.
I'll create a PR ASAP

--- a/utils/aeromatic++/types.cpp
+++ b/utils/aeromatic++/types.cpp
@@ -200,7 +200,7 @@ std::string Param::get_unit(bool upper, unsigned utype, bool convert)
 {
     std::string str = _cvt_t[utype].name[convert];
     if (upper) {
-        std::transform(str.begin(), str.end(), str.begin(), std::toupper);
+        std::transform(str.begin(), str.end(), str.begin(), (int (*)(int))std::toupper);
     }
     return str;
 }

--- a/utils/aeromatic++/types.cpp
+++ b/utils/aeromatic++/types.cpp
@@ -200,7 +200,9 @@ std::string Param::get_unit(bool upper, unsigned utype, bool convert)
 {
     std::string str = _cvt_t[utype].name[convert];
     if (upper) {
-        std::transform(str.begin(), str.end(), str.begin(), (int (*)(int))std::toupper);
+        std::transform(str.begin(), str.end(), str.begin(),
+                       [](unsigned char c){ return std::toupper(c); }
+                      );
     }
     return str;
 }

--- a/utils/aeromatic++/types.cpp
+++ b/utils/aeromatic++/types.cpp
@@ -25,6 +25,7 @@
 #include <iostream>
 #include <sstream>
 #include <string>
+#include <cctype>
 #include <stdlib.h>
 
 #include "types.h"
@@ -199,9 +200,7 @@ std::string Param::get_unit(bool upper, unsigned utype, bool convert)
 {
     std::string str = _cvt_t[utype].name[convert];
     if (upper) {
-        std::transform(str.begin(), str.end(), str.begin(),
-                       [](unsigned char c){ return std::toupper(c); }
-                      );
+        std::transform(str.begin(), str.end(), str.begin(), std::toupper);
     }
     return str;
 }

